src/stores/Frame/RegionSetStore.ts

```diff
@@ -39,12 +40,17 @@ export class RegionSetStore {
             const cursorRegion = this.regions[0];
             // Need to avoid redundant update (position not changed), backend may not reply to redundant requests.
             const roundedPos = {x: Math.round(pos.x), y: Math.round(pos.y)};
-            if (cursorRegion?.regionId === CURSOR_REGION_ID && (cursorRegion.center?.x !== roundedPos.x || cursorRegion.center?.y !== roundedPos.y)) {
+            if (cursorRegion?.regionId === CURSOR_REGION_ID && (!this.isHoverImage || cursorRegion.center?.x !== roundedPos.x || cursorRegion.center?.y !== roundedPos.y)) {
                 cursorRegion.setCenter(roundedPos);
+                this.setIsHover(pos.x + 0.5 >= 0 && pos.x + 0.5 <= this.frame.frameInfo.fileInfoExtended.width && pos.y + 0.5 >= 0 && pos.y + 0.5 <= this.frame.frameInfo.fileInfoExtended.height);
```

Comment:
Perhaps we can change it to `this.setIsHover(true)` because we call this function only when the cursor is on the image.